test/fixtures/es2017/async-functions/44/options.json

```diff
@@ -0,0 +1,3 @@
+{
+  "throws": "Unexpected token, expected ; (1:6)"
```

Comment:
I'm still unsure how we can safely warn insert a better error message for this case, given that this needs to be allowed: https://github.com/babel/babylon/blob/master/test/fixtures/esprima/es2015-identifier/valid_await/actual.js